onboarding_documentation/technical_documentation/metamist_docker/metamist_docker_tutorial.md

```diff
@@ -0,0 +1,737 @@
+
+        # read the output out into GCP
+        # The helper method output_path() will create a new path based on the current project,
+        # test/main, and the output prefix provided to analysis_runner
+        # -o my_output
+        # --dataset my-dataset
+        # --access_level test
+        # output_path('this_file.txt')
+        # -> gs://cpg-my-dataset-test/my_output/this_file.txt
+        b.write_output(
+            j.out_fastqe, output_path(f'{sg}.html', category='web')
+        )  # category='web' writes it to the web bucket
+    b.run(wait=False)
+
+    # endregion
+```
+
+</details>
+<br>
+
+Note the `j.storage('1Gi')` line. We use this to ensure that the job has enough storage to run. Good to know it is a parameter we can adjust!
+
+The code up to this point is capable of fetching the necessary fastq files, running FastQE on them, and writing the output to the web bucket. However, if we were to run this code now, we would be the only ones who know that this analysis has run. We next need to update Metamist with the location of the output files so that other users can access them!
+
+We have not yet covered how to update Metamist with the location of the output files. This is because it is a little more complicated than the previous steps. We will cover this in the next section.
+
+## 4. Update Metamist with the location of the output files
+
+We will be using the metamist `AnalysisApi` endpoint to update the analyses of the `sequencingGroups` we have run FastQE on. Additionally, we will be updating the api endpoint from within Hail Batch which can add complexity. This part is tricky so don't be worried if you need to peek at the answers!!
+
+Firslty, lets create a function that performs the updating. For an example of how to update an analysis endpoint see [here](https://github.com/populationgenomics/automated-interpretation-pipeline/blob/2fbb058d728bf12d82c6c4b14c7c118a64ab8dd4/reanalysis/metamist_registration.py#L20C6-L20C6), focus on line 46 within `register_html`.
+
+The function we'll be writing will take the `SequencingGroup` ID as an argument and will update the `Analysis` of that `SequencingGroup` with the location of the output files. Within this function we will importing the necessary packages and functions as well as getting the necessary information from the config file.
+
+- Tips:
+  - `from cpg_utils.config import get_config` will give us access to the config file from which we can access via dictionary notation. For example, `get_config()['workflow']['dataset']` will return the name of the project we are running the analysis on.
+  - The web bucket is a special bucket that allows us to host html files so they are directly visible in the browser, this bucket is located at `get_config()['storage']['default']['web_url']`. This link is what we will display in the `meta` field of the analysis.
+
+<br>
+<br>
+
+<details>
+<summary>Click to reveal function</summary>
+
+```python
+def create_analysis_entry(
```

Comment:
This and the following code block also probably exist in the other script, so they could be removed here